api-specificatie/openapi.yaml

```diff
@@ -385,6 +391,8 @@ components:
     Adres:
```

Comment:
Het lijkt me goed de parameter identificatie te wijzigen naar: 
    adresseerbaarObjectIdentificatie:
      description: Identificatie van een adresseerbaarobject uit de BAG. Dit kan zijn een verblijfsobject, ligplaats of standplaats. 
      name: identificatie
      in: path
      required: true
      schema:
        title: identificatie van verblijfsobject
        type: string
        pattern: '^[0-9]{4}01[0-9]{10}|[0-9]{4}02[0-9]{10}|[0-9]{4}03[0-9]{10}$'
        example: 1234010000000001, 1234020000000001, 1234030000000001

Reply:
Aangepast in de geest van dit voorstel (Title heb ik "identificatie van een adresseerbaar object van gemaakt.